lib/interferon.rb

```diff
@@ -188,17 +235,15 @@ def create_alerts(dest, alerts_queue)
             while name = alerts_to_create.shift
               break if @request_shutdown
               cur_alert, people = alerts_queue[name]
-
               log.debug "creating alert for #{cur_alert[:name]}"
-              alert_key = dest.create_alert(cur_alert, people)
-              alert_keys << alert_key
+              alert_key_ids << dest.create_alert(cur_alert, people)
             end
           end
           t.abort_on_exception = true
           t
         end
         threads.map(&:join)
-        alert_keys
+        alert_key_ids
```

Comment:
This should move one line below to the end of the method, to ensure it always returns a value.

I have a separate unmerged PR https://github.com/airbnb/interferon/pull/9 for this one-line change, but since you have this PR now, would you mind including this one-line change in your PR? Appreciate it.